app/models/request.rb

```diff
@@ -1,10 +1,12 @@
 # frozen_string_literal: true
 
 class Request < ApplicationRecord
-  belongs_to :point_person, class_name: 'Techie'
+  belongs_to :point_person, class_name: 'Techie', optional: true
   belongs_to :requester, class_name: 'Person'
 
-  scope :open, -> { where('point_person_id is null') }
+  scope :open, -> { where(point_person_id: nil) }
+  scope :assigned, -> { where('point_person_id is not null') }
```

Comment:
We should try to avoid single string arguments in AR queries if doing it with the other types of args isn't too painful.

```suggestion
  scope :assigned, -> { where.not(point_person_id: nil) }
```